cielim.uproject

```diff
@@ -13,28 +13,32 @@
 			]
 		}
 	],
-	"PreBuildSteps":
-	{
-    	"Mac": [
-			"export PATH=/opt/homebrew/bin:$PATH",
-        	"tcsh $(ProjectDir)\/Source\/ThirdParty\/ProtobufLibrary\/build-protobuf-library.sh \"$(ProjectDir)\/Source\/ThirdParty\/ProtobufLibrary\/\" ",
-			"tcsh $(ProjectDir)\/Source\/ThirdParty\/ZMQ\/download-zmq-and-cppzmq.sh \"$(ProjectDir)\/Source\/ThirdParty\/ZMQ\/\" ",
-			"tcsh $(ProjectDir)\/Source\/ThirdParty\/OpenCV\/download-and-build-opencv.sh \"$(ProjectDir)\/Source\/ThirdParty\/OpenCV\/\" "
-    	]
-	},
 	"Plugins": [
 		{
 			"Name": "ModelingToolsEditorMode",
 			"Enabled": true,
 			"TargetAllowList": [
 				"Editor"
 			]
+		},
+		{
+			"Name": "GeometryScripting",
+			"Enabled": true
 		}
 	],
 	"TargetPlatforms": [
 		"Linux",
 		"LinuxArm64",
 		"Mac",
 		"Windows"
-	]
+	],
+	"PreBuildSteps":
+	{
+		"Mac": [
+			"export PATH=/opt/homebrew/bin:$PATH",
+			"tcsh $(ProjectDir)/Source/ThirdParty/ProtobufLibrary/build-protobuf-library.sh \"$(ProjectDir)/Source/ThirdParty/ProtobufLibrary/\" ",
+			"tcsh $(ProjectDir)/Source/ThirdParty/ZMQ/download-zmq-and-cppzmq.sh \"$(ProjectDir)/Source/ThirdParty/ZMQ/\" ",
+			"tcsh $(ProjectDir)/Source/ThirdParty/OpenCV/download-and-build-opencv.sh \"$(ProjectDir)/Source/ThirdParty/OpenCV/\" "
+		]
+	}
```

Comment:
:small_red_triangle_down: single new line at end of file needed.

Reply:
Added a new line!